docs/install/install-staging-new.md

```diff
@@ -0,0 +1,485 @@
+    * Navigate to the root of your ISLE project
+    * `cd /opt/yourprojectnamehere`
+  * `docker-compose pull`
+
+---
+
+### Step 14: On Remote Staging - Start Containers
+
+**Please note:** Prior to starting the launch process, it is recommended that you briefly open your firewall to allow ports 80 and 443 access to the world. You'll only need to keep this open for 3 -5 minutes and then promptly close access once the Let's Encrypt SSL certificates have been generated.
+
+* _Using the same open terminal / Powershell_
+  * `docker-compose up -d`
+
+* Please wait a few moments for the stack to fully come up. Approximately 3-5 minutes.
+
+* After the above process is completed using the already open terminal or Powershell again.
+    * View only the running containers: `docker ps`
+    * View all containers (both those running and stopped): `docker ps -a`
+    * All containers prefixed with `isle-` are expected to have a `STATUS` of `Up` (for x time).
+      * **If any of these are not `UP`, then use [ISLE Installations: Troubleshooting](install-troubleshooting.md) to solve before continuing below.**
+      <!---TODO: This could be confusing if (a) there are other, non-ISLE containers, or (b) the isle-varnish container is installed but intentionally not running, or (c) older exited ISLE containers that maybe should be removed. --->
+
+* In your web browser, enter your Staging site URL: `https://yourprojectnamehere-staging.institution.edu`
+  * **Please note:** You should not see any errors with respect to the SSL certifications, you should see a nice green lock padlock for the site security. If you see a red error or unknown SSL cert provider, you'll need to shut the containers down and review the previous steps taken especially if using Let's Encrypt. You may need to repeat those steps to get rid of the errors.
+
+---
+
+### Step 15: On Remote Staging - Import the Local MySQL Drupal database
+
+Prior to attempting this step, do consider the following:
+
+* If the end user is running multi-sites, there will be additional databases to export.
+
+* Do not import the `fedora3` database
+
+---
+
+#### Local Drupal site database import process
+
+* Copy the `local_drupal_site_082019.sql` created in Step 1 to the Remote Staging server.
+
+* Import the exported `Local` MySQL database for use in the current `Staging` Drupal site. Refer to your `staging.env` for the usernames and passwords used below.
+  * You can use a MySQL GUI client for this process instead but the command line directions are only included below.
+  * Run `docker ps` to determine the MySQL container name
+  * Shell into your currently running `Staging` MySQL container
+    * `docker exec -it yourmysql-container-name bash`
+  * Import the Local Drupal database. Replace the `DRUPAL_DB` & `DRUPAL_DB_USER` below in the command with the values found in your `staging.env`.
+    * `mysqldump -u DRUPAL_DB_USER -p DRUPAL_DB < local_drupal_site_082019.sql`
```

Comment:
I believe this needs to be the root user, as the DRUPAL_DB_USER doesn't exist yet.

Reply:
By default spinning up a blank staging system will give you a blank / empty database that you'll then import the Production (non-ISLE) database into. This action can be preformed by the DRUPAL_DB_USER which will exist once the ISLE container init process has completed on start up. So no the root user doesn't need to be involved here necessarily.

Reply:
ah, actually the issue is that should be `mysql` not `mysqldump`, I think.

Reply:
Yes you are right. I'll update and push.

Reply:
Thanks for spotting this.

Reply:
Fixed in this commit. https://github.com/Islandora-Collaboration-Group/ISLE/pull/287/commits/a5e759b06ed49af9f0e737f4b501cf5d558bbc06